### PR TITLE
add docker support

### PR DESCRIPTION
**Pull request by ihorromaniuk**

### CODE REVIEW

**Review by sarakhmen — CHANGES_REQUESTED** [review]
Have you tested your solution properly? Does your docker-compose up really work? I see u use 5.7 version of MySQL server and it probably causes some issues during startup

**Comment by ihorromaniuk** [comment]
> Have you tested your solution properly? Does your docker-compose up really work? I see u use 5.7 version of MySQL server and it probably causes some issues during startup

image

**Comment by ihorromaniuk** [comment]
And endpoints work as intendent

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ports:
+      - $SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT
+      - $DEBUG_PORT:$DEBUG_PORT
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url"  : "jdbc:mysql://mysql:$MYSQL_DOCKER_PORT/$MYSQL_DATABASE",
+        "spring.datasource.username" : "$MYSQL_USER",
+        "spring.datasource.password" : "$MYSQL_PASSWORD"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (liliia-ponomarenko) [inline docker-compose.yml]: Let’s  use DEBUG_PORT variable

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -1,17 +1,12 @@
 spring:
   application:
     name: bookstore
-  datasource:
-    driver-class-name: com.mysql.cj.jdbc.Driver
-    url: jdbc:mysql://localhost:3306/bookstore?serverTimezone=UTC
-    username: book_user
-    password: book1234
```
Comment (liliia-ponomarenko) [inline src/main/resources/application.yml:8]: Don’t remove properties from the configuration file. Let’s  use variables from .env

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+version: "3.8"
+
+services:
+  mysql:
+    image: mysql:5.7
```
Comment (liliia-ponomarenko) [inline docker-compose.yml]: it is better to use the latest versions of technologies

**Review by liliia-ponomarenko — CHANGES_REQUESTED** [review]
Good job! Let’s fix a few comments ;)

**Review by sarakhmen — APPROVED** [review]

**Pull request merged**